src/scss/theme/banners/_banner-in-page--testimonial-image-and-text.scss

```diff
@@ -1,53 +1,90 @@
 @charset "UTF-8";
 
 $banner-circle--image-width: 200px;
-$banner-circle--padding: 20px;
 
 // only apply to banners inside the react paragraph page content section
 .paragraph-item {
-
   // if there is an image and a card  - testimonial banner
   .saa-banner--has-image {
     .su-hero {
+      background: $su-color-light-sandstone;
+      height: auto;
+      min-height: unset;
+      position: unset;
+      padding-top: 72px;
+      padding-bottom: 108px;
+
       .su-hero__media {
-        background: $su-color-white;
+        background: transparent;
+        min-height: unset;
+        position: unset;
 
-        img {
+        img { // Image display has been set to media image style - Medium (220)
+          border-radius: 50%;
+          width: $banner-circle--image-width;
+          height: $banner-circle--image-width;
           max-width: $banner-circle--image-width;
```

Comment:
The `max-width/max-height` properties are no longer needed but I left them just in case — would there be a reason why we need to keep them?  🤔

Reply:
We can go ahead and get rid of max-width and max-height since we are setting it :)